### PR TITLE
divest bvecvx to vebal

### PR DESCRIPTION
**Pull request by jalbrekt85**
handles #461

### CODE REVIEW

File: scripts/issue/461/buy_bal_and_lock.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    trops = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    trops.init_cow()
+
+    cvx = trops.contract(registry.eth.treasury_tokens.CVX)
+    bal = trops.contract(registry.eth.treasury_tokens.BAL)
+
+    trops.cow.allow_relayer(cvx, cvx.balanceOf(trops))
+    half = int(cvx.balanceOf(trops) / 2)
+    a_tenth = int(cvx.balanceOf(trops) / 10)
+    rest = cvx.balanceOf(trops) - half - 4 * a_tenth
```
Comment (jorijnsmit) [inline scripts/issue/461/buy_bal_and_lock.py:47]: `cvx.balanceOf(trops)` isnt necessarily equal to 25k

**Review by jorijnsmit — COMMENTED** [review]
any way we can get the first couple of steps merged already, so we can get started with posting them?

File: scripts/issue/461/buy_bal_and_lock.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    weth = vault.contract(registry.eth.treasury_tokens.WETH)
+
+    trops.take_snapshot(tokens=[bal])
+    vault.take_snapshot(tokens=[vebal, weth, bal.address])
+
+    bal.transfer(vault, bal.balanceOf(trops))
+
+    bal = vault.contract(registry.eth.treasury_tokens.BAL)
+
+    vault.balancer.lock_bal(mantissa_bal=bal.balanceOf(vault))
```
Comment (jorijnsmit) [inline scripts/issue/461/buy_bal_and_lock.py:81]: this wont work, tx can only be posted from one multisig at a time. so will need its own func im afraid.

**Review by jorijnsmit — CHANGES_REQUESTED** [review]
just need to isolate that last action of locking to its own method.

File: scripts/issue/461/buy_bal_and_lock.py (hunk 2 above)
Comment (jalbrekt85) [reply scripts/issue/461/buy_bal_and_lock.py]: good point

**Review by petrovska-petro — APPROVED** [review]
Series of actions aligns with the treasury decision outlined, code looks good!

@jalbrekt85 you nailed the balancer class, seems that not methods are missing to interact with the protocol 🔥

**Review by jorijnsmit — APPROVED** [review]

**Comment by jorijnsmit** [comment]
merging and posting first step

**Pull request merged**